cluster/manifests/skipper/daemonset.yaml

```diff
@@ -28,11 +28,13 @@ spec:
       hostNetwork: true
       containers:
       - name: skipper-ingress
-        image: registry.opensource.zalan.do/teapot/skipper:v0.1.1
+        image: registry.opensource.zalan.do/pathfinder/skipper:v0.9.8a
         ports:
         - name: ingress-port
           containerPort: 9999
           hostPort: 9999
+        command:
+          - /usr/bin/skipper
```

Comment:
any change of having this fixed in skipper Dockerfile instead so we don't need to specify the command ?